ebssurrogate/files/apparmor_profiles/usr.lib.postgresql.bin.postgres

```diff
@@ -0,0 +1,29 @@
+#include <tunables/global>
+
+profile /usr/lib/postgresql/bin/postgres flags=(attach_disconnected) {
+#include <abstractions/base>
+#include <abstractions/bash>
+#include <abstractions/consoles>
+#include <abstractions/nameservice>
+#include <abstractions/openssl>
+#include <abstractions/ssl_keys>
+#include <abstractions/user-tmp>
+
+/dev/shm rw,
+deny @{HOME}/** rwx,
+/etc/postgresql-custom/** r,
+/etc/postgresql/** r,
+/run/systemd/notify rw,
+/var/run/systemd/notify rw,
+/usr/bin/cat rix,
+/usr/bin/dash rix,
+/usr/bin/mknod rix,
+/usr/lib/postgresql/bin/* mrwix,
```

Comment:
write needed?

Reply:
(at a more general level, trying to apply w^x)

Reply:
let me remove this too.